rating.jenkins.io.tf

```diff
@@ -12,3 +12,16 @@ resource "postgresql_role" "rating" {
   login    = true
   password = random_password.pgsql_rating_user_password.result
 }
+
+output "dbconfig" {
```

Comment:
```suggestion
# This (sensitive) output is meants to be encrypted into the production secret system, to be provided as a secret to the ratings.jenkins.io application
output "rating_dbconfig" {
```

Reply:
Suggestion to add:
- A comment (as per @timja comment) explaining the role of the output
- rename the output by adding "rating" as prefix, since the output namespace is global to the whole repository